app/moves/views/_layout.njk

```diff
@@ -23,7 +23,9 @@
   {% block pageHeader %}
     <header class="govuk-grid-row">
       <div class="{% block headerGridColumnClass %}govuk-grid-column-two-thirds{% endblock %}">
-        <span class="govuk-caption-xl">{{ t(pageTitle) }}</span>
+        <span class="govuk-caption-xl">{{ t(pageTitle, {
```

Comment:
For the purposes of this PR this looks great, but is there any reason why we couldn't move the caption inside the h1? eg.  

```
<h1 class="govuk-!-margin-bottom-2">
  <span class="govuk-caption-xl">Upcoming moves for</span>
  <time class="govuk-heading-xl" datetime="2020-04-13">
    Monday 13 Apr 2020
  </time>
</h1>
```

Reply:
There isn't. I like that suggestion, particularly the addition of the `time` attribute. The use of `time` is something I wanted to review actually and use more consistently.

We have it in a few places but not in all the places we use dates/times.

I'll create a GitHub issue to address this as a separate change :)

Reply:
@solidgoldpig In fact, the reason was because that's how the example is given in the [GOV.UK Design System](https://design-system.service.gov.uk/styles/typography/#headings-with-captions).

But maybe it depends on the content and context. I think you might be right that in this instance that it would read better if the caption was inside and therefore part of the heading as opposed to outside of it. What do you think?

Reply:
Have created #428 to ensure we use `time` wherever possible.

Reply:
I think it is part of the page heading. And lo... the design system covers this very scenario

> If the caption should be considered part of the page heading, you can also nest the caption within the H1.

```html
<h1 class="govuk-heading-xl">
  <span class="govuk-caption-xl">govuk-caption-xl</span>
  govuk-heading-xl
</h1>
```